packages/app/obojobo-express/config/permission_groups.json

```diff
@@ -4,7 +4,8 @@
 		"canEditDrafts": ["Instructor", "Administrator", "urn:lti:instrole:ims/lis/Administrator", "urn:lti:role:ims/lis/TeachingAssistant"],
 		"canDeleteDrafts": ["Instructor", "Administrator", "urn:lti:instrole:ims/lis/Administrator", "urn:lti:role:ims/lis/TeachingAssistant"],
 		"canCreateDrafts": ["Instructor", "Administrator", "urn:lti:instrole:ims/lis/Administrator", "urn:lti:role:ims/lis/TeachingAssistant"],
-		"canViewDrafts": ["Instructor", "Administrator", "urn:lti:instrole:ims/lis/Administrator", "urn:lti:role:ims/lis/TeachingAssistant"]
+		"canViewDrafts": ["Instructor", "Administrator", "urn:lti:instrole:ims/lis/Administrator", "urn:lti:role:ims/lis/TeachingAssistant"],
+		"canCompleteAssignment": ["Learner", "urn:lti:role:ims/lis/Learner"]
```

Comment:
The name for this trips me up a little, because it's really 'canViewTheActualModuleNotPreviewMode` (but of course we can't call it that). Because if this is false then they are forced into preview mode. And technically they _can_ complete the assignment, it's just that we probably can't send a score to any gradebook via LTI.

Also, and this isn't in regards to this PR, there is a 'canViewDrafts' perm which doesn't include Learner, which seems confusing since Learners can view drafts.

So one idea: Add Learner to canViewDrafts and re-evaulate how that perm is being used. Then perhaps add a `canOnlyPreviewDrafts` which would contain everyone that canViewDrafts but exclude Learner.

Not sure I like it. Tossing out some other name ideas: `canSubmitAssessment`, `canSubmitAttempt`, `canSendLTIScore`.

Reply:
we landed on `canViewAsStudent`

Reply:
I renamed canViewDrafts to canPreviewDrafts.

It was pretty much used the same way, so I think it works.